modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsDateAndTime.testcase

```diff
@@ -129,5 +129,53 @@ definition {
 		}
 	}
 
+	@description = "LPS-143068 - Verify a custom object entry with a date and time field can be updated"
+	@priority = 5
+	test CanUpdateEntry {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a date and time field and an object entry") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 186247",
+				objectName = "CustomObject186247",
+				pluralLabelName = "Custom Objects 186247");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "DateTime",
+				fieldLabelName = "Custom Date Time",
+				fieldName = "customDateTime",
+				fieldType = "DateTime",
+				isRequired = "false",
+				objectName = "CustomObject186247",
+				timeStorageOption = "convertToUTC");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186247");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customDateTime",
+				objectName = "CustomObject186247",
+				value = "2023-06-01T12:00:00.000Z");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject186247");
```

Comment:
This should be part of the `When`, it's not part of the setup to create the object and the entry